canonical/ubuntu.com
# Update VMware webinar engage page links

## Done

- Added a redirect from /engage/vmware-to-charmed-openstack to /engage/from-vmware-to-open-source
- Updated several pages to reflect changes associated with this redirect

## QA

- Visit https://ubuntu-com-12302.demos.haus/engage/vmware-to-charmed-openstack, see that you are redirected to https://ubuntu-com-12302.demos.haus/engage/from-vmware-to-open-source
- Visit the following pages and see that the changes match the copy changes from around October 25th:
  - https://ubuntu-com-12302.demos.haus/openstack ([copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit))
  - https://ubuntu-com-12302.demos.haus/openstack/features ([copy doc](https://docs.google.com/document/d/1odFXuFtN1GjuZvgd8tW48HB4J9PjZJtP3w6wY0xlmog/edit))
  - https://ubuntu-com-12302.demos.haus/openstack/consulting ([copy doc](https://docs.google.com/document/d/1f83OSIHVQGMjF7b0pZUKFvoubMGqi_b-OYbSpyhveX4/edit#))
  - https://ubuntu-com-12302.demos.haus/server ([copy doc](https://docs.google.com/document/d/1vxLhtbjgPs9PCNEhr7fbhEoErCPMXYFfU4UTKH7Kbq4/edit))
  - https://ubuntu-com-12302.demos.haus/support ([copy doc](https://docs.google.com/document/d/1AU2dwTXpQk2UuEvSuAqRKrbPWYb-LR0ZGvWu6xw95I8/edit#))
  - https://ubuntu-com-12302.demos.haus/managed ([copy doc](https://docs.google.com/document/d/19GoMudyrd9Zt1e7kfj-koL-Lt0RGYzWHRPuuKYUPvqw/edit))
  - https://ubuntu-com-12302.demos.haus/lxd ([copy doc](https://docs.google.com/document/d/1LJ6ajHYAmKdxSNDaD5kbriDod1t7RcmIwVJtevAeKFQ/edit#))

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6145
